spec/core/player_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rspec'
+require 'UI/UI'
+require 'core/player'
+require 'core/board'
+
+describe Player do
+
```

Comment:
In rspec, you should use let (https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let) to handle variables across tests.

so let(:board) { Board.new(3) } and so on.